nncf/tensorflow/quantization/algorithm.py

```diff
@@ -130,7 +132,22 @@ def _create_quantizer(self, name: str, qspec: TFQuantizerSpec) -> Quantizer:
         return quantizer_cls(name, qspec)
 
     def get_transformation_layout(self, model: tf.keras.Model) -> TFTransformationLayout:
-        nncf_graph = convert_keras_model_to_nncf_graph(model)
+        converter = TFModelConverterFactory.create(model)
+        nncf_graph = converter.convert()
+
+        # Steps to enable prop-based quantization:
```

Comment:
Could you add step by step instructon to the relevant ticket and remove here?

Reply:
Since it is already implemented in #803, will just remove it.